remmina/__init__.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+"""Open Remmina remote connections
+
+Synopsis: <trigger> <query>"""
+
+from albertv0 import *
+import os
+from os.path import expanduser
+from pathlib import Path
+
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Remmina"
+__version__ = "1.0"
+__trigger__ = "re "
+__author__ = "gentoo90"
+__dependencies__ = ["remmina"]
+
+remmina_icon_path = iconLookup("org.remmina.Remmina")
+
+protocol_icons = {
+    "EXEC": iconLookup("remmina-tool-symbolic"),
+    "NX": iconLookup("remmina-nx-symbolic"),
+    "RDP": iconLookup("remmina-rdp-symbolic"),
+    "SFTP": iconLookup("remmina-sftp-symbolic"),
+    "SPICE": iconLookup("remmina-spice-symbolic"),
+    "SSH": iconLookup("remmina-ssh-symbolic"),
+    "VNC": iconLookup("remmina-vnc-symbolic"),
+    "XDMCP": iconLookup("remmina-xdmcp-symbolic")
+}
+
+connections = []
+
+
+class RemminaConnection:
+    def __init__(self, path):
+        self.path = path
+        self.name = str(self.path)
+        self.icon = remmina_icon_path
+
+        with open(self.path) as f:
+            for l in f.readlines():
+                if l.startswith("name"):
+                    self.name = l.split("=")[1].strip()
+                if l.startswith("protocol"):
+                    protocol = l.split("=")[1].strip()
+                    try:
+                        self.icon = protocol_icons[protocol]
+                    except KeyError:
+                        pass
+
+    def __repr__(self):
+        return f"{self.name} ({self.path})"
+
+
+def initialize():
+    conf_dir = Path(expanduser("~/.remmina"))
+    debug(f"CONF DIR: {conf_dir}")
+    global connections
+    connections = [RemminaConnection(p) for p in conf_dir.glob("*.remmina")]
```

Comment:
I wouldn't initialise the connections in `initialize`.
This is called only when the plugin is loaded by albert (e.g., when the user has ticked the plugin in Albert's settings.
Thus, connections added *after* the plugin has been loaded will not be detected.

Since this isn't a runtime operation, just do it  at the beginning of `handleQuery`